errors/minification-disabled.md

```diff
@@ -0,0 +1,18 @@
+# Minification disabled in production
+
+#### Why This Error Occurred
+
+When overriding `optimization.minimize` or `optimization.minimizer` incorrectly in `next.config.js`, this degrades performance of your application
```

Comment:
```suggestion
Code optimization has been disabled for your **production build**.
The `optimization.minimize` or `optimization.minimizer` was incorrectly overridden in `next.config.js`.

This severely degrades your application's performance at runtime. It can also result in server-side-only code being downloaded by your users.
```